packages/docs-site/src/content/docs/resources/developer-tools.mdx

```diff
@@ -18,6 +18,17 @@ import { LinkCard, CardGrid } from "@astrojs/starlight/components";
     description="A Dune dashboard to track Taiko BCR mainnet, built by Taiko Labs."
     href="https://dune.com/taiko_xyz/taikobcr"
   />
+  <LinkCard
+    title="SocialScan Explorer - Mainnet"
+    description="A Mainnet explorer powered by SocialScan (Hemera)"
+    href="https://taiko.socialscan.io"
```

Comment:
Duplicate of https://taikoscan.net/

Reply:
Actually we should bring all explorers listed below:

![Uploading Screenshot 2024-12-17 at 9.38.53 AM.png…]()